src/common/framework/test_group.ts

```diff
@@ -278,7 +279,7 @@ class RunCaseSpecific<
       let totalCount = 0;
       let skipCount = 0;
       for (const subParams of this.subParamGen(this.params)) {
-        rec.info(new Error('subcase: ' + stringifyPublicParams(subParams)));
+        rec.info(new Error('subcase: ' + stringifyPublicParamsUniquely(subParams)));
```

Comment:
nit: I think this is a merge error with #454?
```suggestion
        rec.info(new Error('subcase: ' + stringifyPublicParams(subParams)));
```